static/app/views/dashboards/datasetConfig/errorsAndTransactions.tsx

```diff
@@ -688,15 +688,16 @@ export function filterAggregateParams(
 }
 
 const shouldSendWidgetForSplittingDiscover = (organization: Organization) => {
-  return organization.features.includes('performance-discover-widget-split-ui');
+  return organization.features.includes('performance-discover-dataset-selector');
 };
 
 const getQueryExtraForSplittingDiscover = (
   widget: Widget,
   organization: Organization,
-  useOnDemandMetrics: boolean
+  _useOnDemandMetrics: boolean
 ) => {
-  if (!useOnDemandMetrics || !shouldSendWidgetForSplittingDiscover(organization)) {
+  // TODO: What's the correct condition here for sending the split?
+  if (!shouldSendWidgetForSplittingDiscover(organization)) {
```

Comment:
```suggestion
  if (!organization.features.includes('performance-discover-dataset-selector')) && (!useOnDemandMetrics || !organization.features.includes('performance-discover-widget-split-ui')) {
  ```
  
  something like this so we don't clobber over the old condition if the dataset selector flag isn't enabled